### DISCUSSION TITLE
Non functioning features

### DISCUSSION THREAD

**Discussion by HughBarstead** [gopro/labs#577] [Q&A]
Hello. I have a Hero 11 running Labs 02.12.70. Great fun playing with all the “FourCC” commands.

I am having trouble with the following commands however. AUDS, CBAR and LBAR do not appear to be functioning. The QR code gets created, the camera reads it, the check mark appears and then nothing. The commands seem to have no effect at all. I am aware of the 4K/30 or less requirements for LBAR and CBAR. AUDS doesn’t list any required conditions. 

Before I go to the extent of resetting everything and reinstalling Labs, can anyone tell me if there are known issues with these commands? Has anyone got any of these features working? 

Thanks in advance for any feedback you can offer.

**Comment by dnewman-gpsw** [comment]
oMAUDS=1 This also required the voice control to be active if you want to see audio levels before capture.

oMLBAR=1oMCBAR=1 Works fine, but it only renders the bars to the capture video, bottom right corner by default.

![image](https://github.com/gopro/labs/assets/10856084/126de06c-5b53-4c94-8e19-ce28c7933e85)

**Reply by HughBarstead** [reply]
Thank you so much for your reply. I was unaware of the need for voice commands to be turned on for AUDS. 

The CBAR and LBAR features are not working for me though. Nothing is recorded in video. 

I guess it’s time to reset and reinstall. At least now I know I’m not banging my head against the wall. Thanks again.

**Reply by dnewman-gpsw** [reply]
Reinstall with not help, you can't get a bad install. Make sure 10-bit is off.

**Reply by HughBarstead** [reply]
Well, nothing was helping with the colour bars and the luminosity sweep, so I did a reset and reinstalled Labs. Now everything works!

I don’t think it was a bad install of the firmware in the first place. I’ve been “playing” a lot and I likely did something stupid.

Thanks for all the help. 100% happy now.
